llm/sglang/llava.yaml

```
@@ -0,0 +1,37 @@
+service:
+  # Specifying the path to the endpoint to check the readiness of the service.
+  readiness_probe: /health
+  # How many replicas to manage.
+  replicas: 2
+
+envs:
+  MODEL_NAME: liuhaotian/llava-v1.6-vicuna-7b
+  TOKENIZER_NAME: llava-hf/llava-1.5-7b-hf
+  # HF_TOKEN: <your-huggingface-token> # Change to your own huggingface token
+  HF_TOKEN: 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
+
+resources:
+  accelerators: {L4:1, A10G:1, A10:1, A100:1, A100-80GB:1}
+  ports:
+    - 8000
+
+setup: |
+  conda activate sglang
+  if [ $? -ne 0 ]; then
+    conda create -n sglang python=3.10 -y
+    conda activate sglang
+  fi
+
+  pip list | grep sglang || pip install "sglang[all]"
+  pip install git+https://github.com/huggingface/transformers
```

Let’s pin the version for transformers for stability.
```suggestion
  pip list | grep transformers || pip install transformers==4.37.2
```